site/en/docs/privacy-sandbox/user-agent/index.md

```diff
@@ -173,13 +173,12 @@ Scaled availability for User-Agent reduction means the fully reduced UA string
 shipped on all Chrome devices. Reduction began with a Chrome
 minor release in Q2 of 2022.
 
+Refer to [User-Agent Strings](/docs/multidevice/user-agent/) for examples of new User-Agent strings.
+
 ### Test the string locally {: #test-locally}
 
-There are a couple of methods to test the reduced User-Agent locally:
+To test the reduced User-Agent locally:
```

Comment:
(this has shipped now... so this section for local testing is less useful. Maybe we can just delete it?)

Reply:
Maybe delete all except do this: 
### Test custom strings locally {: #test-locally}

If you want to test your site using custom User-Agent strings to simulate
different devices, one way is to launch Chrome with the
`--user-agent="Custom string here"` command-line flag. Find more on
[command-line flags][10] here.